usagov/vote-gov-drupal
# Qa/vote 2821 cypress image update

## Jira ticket

[Vote-2821](https://cm-jira.usa.gov/browse/VOTE-2821)

## Description

Updated yml file for updated cypress version for both frontend and link validation tests 

Removed .nvmrc file that is not needed in testing folder

## Deployment and testing

### Post-deploy steps

1. n/a

### QA/Testing instructions

1. Review file and ensure `cypress 13.4.2` is added in the docker image 
2. [Review test run](https://app.circleci.com/pipelines/github/usagov/vote-gov-drupal/95906/workflows/98f118ca-8bfe-4a55-8315-c033796657ec/jobs/103441) to verify that tests are passing in the pipeline

## Checklist for the Developer




## Checklist for the Peer Reviewers
